### PR TITLE
Fixed #30923 -- updated example of rendering template on custom template tags docs

### PR DESCRIPTION
**Pull request by seanbermejo**

### CODE REVIEW

File: docs/howto/custom-template-tags.txt [hunk 1]
```diff
@@ -794,8 +794,10 @@ The ``__init__`` method for the ``Context`` class takes a parameter called
 This is not a very common situation, but it's useful if you're rendering a
 template yourself. For example::
 
+    from django.template import Context
+
     def render(self, context):
-        t = context.template.engine.get_template('small_fragment.html')
+        t = context.template.engine.get_template('small_fragment.html').template
```
Comment (felixxm) [inline docs/howto/custom-template-tags.txt:800]: `Template` has not `template` attribute.

**Review by felixxm — COMMENTED** [review]
@seanbermejo Thanks for this patch :+1: However it seems these examples work as is. I'm going to close the ticket.

**Comment by felixxm** [comment]
ticket-30923

File: docs/howto/custom-template-tags.txt (hunk 1 above)
Comment (seanbermejo) [reply docs/howto/custom-template-tags.txt:800]: @felixxm Sorry about that, I was testing it on `Django=3.2.7`. Thanks for your review!

**Pull request closed without merging**